.github/new-prs-labeler.yml

```diff
@@ -89,6 +119,16 @@ compiler-rt:scudo:
   - compiler-rt/lib/scudo/**
   - compiler-rt/test/scudo/**
 
+compiler-rt:tsan:
+  - compiler-rt/lib/tsan/**
```

Comment:
There are also files that correspond to each of sanitizers in compiler-rt/include/sanitizer/